utility/build.gradle

```diff
@@ -62,4 +62,6 @@ dependencies {
   kaptTest(
       'com.google.dagger:dagger-compiler:2.24'
   )
+
+  implementation project(":model")
```

Comment:
This dependency seems problematic. Doesn't model depend on util? I'd expect this to create a circular dependency.

Alas, another side effect of Gradle. :( #59 can't come soon enough.